docs/en/reference/limitations-and-known-issues.rst

```diff
@@ -130,10 +130,49 @@ included in the core of Doctrine ORM. However there are already two
 extensions out there that offer support for Nested Set with
 ORM:
 
-
 -  `Doctrine2 Hierarchical-Structural Behavior <https://github.com/guilhermeblanco/Doctrine2-Hierarchical-Structural-Behavior>`_
 -  `Doctrine2 NestedSet <https://github.com/blt04/doctrine2-nestedset>`_
 
+Using Traits in Entity Classes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There is no real "official" statement whether or to which extent traits
+are supported in entity or mapped superclasses.
```

Comment:
But since this is the official documentation, we're about to make an official statement, aren't we? I find this paragraph irritating.

Reply:
You’re right.

So, what can we say?

“For the time being, traits are not officially supported.” ?